expr/func_scaletoseconds.go

```diff
@@ -0,0 +1,65 @@
+package expr
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+
+	"github.com/grafana/metrictank/api/models"
+	schema "gopkg.in/raintank/schema.v1"
+)
+
+type FuncScaleToSeconds struct {
+	in      GraphiteFunc
+	seconds float64
+}
+
+func NewScaleToSeconds() GraphiteFunc {
+	return &FuncScaleToSeconds{}
+}
+
+func (s *FuncScaleToSeconds) Signature() ([]Arg, []Arg) {
+	return []Arg{
+		ArgSeriesList{val: &s.in},
+		ArgFloat{key: "seconds", val: &s.seconds},
+	}, []Arg{ArgSeriesList{}}
+}
+
+func (s *FuncScaleToSeconds) Context(context Context) Context {
+	return context
+}
+
+func (s *FuncScaleToSeconds) Exec(cache map[Req][]models.Series) ([]models.Series, error) {
+	series, err := s.in.Exec(cache)
+	if err != nil {
+		return nil, err
+	}
+
+	out := make([]models.Series, len(series))
```

Comment:
Use the pointSlicePool?

Reply:
I thought that's for `[]schema.Point` not `[]models.Series`

Reply:
Yeah, sorry I missed that you used it below.